osgp/platform/osgp-core/src/main/java/org/opensmartgridplatform/core/infra/jms/protocol/incoming/AbstractProtocolRequestMessageProcessor.java

```diff
@@ -5,7 +5,7 @@
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
-package org.opensmartgridplatform.core.infra.jms.protocol.in;
+package org.opensmartgridplatform.core.infra.jms.protocol.incoming;
```

Comment:
maybe the package should be named org.opensmartgridplatform.core.infra.jms.protocol.inbound